style.css

```diff
@@ -91,13 +96,6 @@ header{
     pointer-events: none;
 }
 
-*::before,
-*::after {
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-}
-
 :root {
```

Comment:
color variables are set on a project level(by the maintainer). Remove these, you don't need it for this issue